<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AAE System Control Dashboard - Metalex 2023</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Dashboard</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QGroupBox" name="groupBox_station_0">
              <property name="title">
               <string>Station 0 : Raw Material</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_groupBox_station_0"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_station_1">
              <property name="title">
               <string>Station 1 : PCB Screwing</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_groupBox_station_1"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_station_2">
              <property name="title">
               <string>Station 2 : PCB Assembly</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_groupBox_station_2"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_station_3">
              <property name="title">
               <string>Station 3 : Automated Optical Inspection</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_groupBox_station_3"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QGroupBox" name="groupBox_mir">
              <property name="title">
               <string>MiR</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_groupBox_mir"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>UR</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>MiR</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Settings</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout">
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Username</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="lineEdit_setting_mir_password">
                 <property name="styleSheet">
                  <string notr="true"> background-color: #f2f2f2;
color: #333; 
padding: 2px;
border: 1px solid #ccc;
border-radius: 2px;
selection-color: black;
selection-background-color: #ccc;</string>
                 </property>
                 <property name="text">
                  <string>distributor</string>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                 <property name="placeholderText">
                  <string>********</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="lineEdit_setting_mir_username">
                 <property name="styleSheet">
                  <string notr="true"> background-color: #f2f2f2;
color: #333; 
padding: 2px;
border: 1px solid #ccc;
border-radius: 2px;
selection-color: black;
selection-background-color: #ccc;</string>
                 </property>
                 <property name="text">
                  <string>distributor</string>
                 </property>
                 <property name="placeholderText">
                  <string>distributor</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Password</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_setting_mir_ip">
                 <property name="styleSheet">
                  <string notr="true"> background-color: #f2f2f2;
color: #333; 
padding: 2px;
border: 1px solid #ccc;
border-radius: 2px;
selection-color: black;
selection-background-color: #ccc;</string>
                 </property>
                 <property name="text">
                  <string>192.168.2.220</string>
                 </property>
                 <property name="placeholderText">
                  <string>192.168.2.222</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>IP Address</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>Response Time (Latency)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_2">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"> background-color: #f2f2f2;
color: #333; 
padding: 2px;
border: 1px solid #ccc;
border-radius: 2px;
selection-color: black;
selection-background-color: #ccc;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="placeholderText">
                  <string>100</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>ms</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>Frequency</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit">
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"> background-color: #f2f2f2;
color: #333; 
padding: 2px;
border: 1px solid #ccc;
border-radius: 2px;
selection-color: black;
selection-background-color: #ccc;</string>
                 </property>
                 <property name="text">
                  <string>1</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>Hz</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="line_10">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_connect_mir">
                 <property name="text">
                  <string>Connect</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Station missions</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="8" column="1">
              <widget class="Line" name="line_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="10" column="2">
              <widget class="QComboBox" name="comboBox_station_3_place"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Station 1 : Screwing</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="Line" name="line_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="9" column="2">
              <widget class="QComboBox" name="comboBox_station_3_pick"/>
             </item>
             <item row="5" column="1">
              <widget class="Line" name="line_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="6" column="2">
              <widget class="QComboBox" name="comboBox_station_2_pick"/>
             </item>
             <item row="2" column="1">
              <widget class="Line" name="line_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Station 0 : Raw material</string>
               </property>
              </widget>
             </item>
             <item row="10" column="1">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Place</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Place</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Pick</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="Line" name="line_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QComboBox" name="comboBox_station_0_pick"/>
             </item>
             <item row="7" column="2">
              <widget class="QComboBox" name="comboBox_station_2_place"/>
             </item>
             <item row="1" column="2">
              <widget class="QComboBox" name="comboBox_station_0_place"/>
             </item>
             <item row="3" column="2">
              <widget class="QComboBox" name="comboBox_station_1_pick"/>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Station 2 : Assembly</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QComboBox" name="comboBox_station_1_place"/>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Pick</string>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Pick</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Place</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Place</string>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Station 3 : Inspection</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="Line" name="line_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="8" column="2">
              <widget class="Line" name="line_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Pick</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFile_save"/>
    <addaction name="actionFile_exit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp_charintorn_autoflexible_com"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFile_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelp_charintorn_autoflexible_com">
   <property name="text">
    <string>charintorn@autoflexible.com</string>
   </property>
  </action>
  <action name="actionFile_save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
